src/components/SkillCardList/SkillCardList.js

```diff
@@ -140,9 +140,12 @@ class SkillCardList extends Component {
 
   render() {
     let skillDisplay = '';
-    if (this.props.skills.length) {
-      skillDisplay = this.state.cards;
+    if (this.props.skills) {
+      if (this.props.skills.length) {
```

Comment:
Just make a small change by using `&&` on the same `if` statement. `&&` short-circuits the condition check, so wouldn't be a problem.